.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseHolidayExtractor.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Recognizers.Text.DateTime
     public class BaseHolidayExtractor : IDateTimeExtractor
     {
         private const string ExtractorName = Constants.SYS_DATETIME_DATE; // "Date";
+        private const string WeekendExtractorName = Constants.SYS_DATETIME_DATEPERIOD; // "Daterange";
```

Comment:
It could be named RangeExtractorName as it can be reused in the future for multi-day holidays. A code comment can explain the difference to the regular ExtractorName.